Makefile

```diff
@@ -29,6 +34,21 @@ tools: $(TOOLS)
 $(TOOLS): oci-%:
 	go build -ldflags "-X main.gitCommit=${COMMIT}" ./cmd/$@
 
+.PHONY: _man install uninstall
+
+_man:
+	go-md2man -in "cmd/oci-unpack/oci-unpack.1.md" -out "oci-unpack.1"
+	go-md2man -in "cmd/oci-image-validate/oci-image-validate.1.md" -out "oci-image-validate.1"
+	go-md2man -in	"cmd/oci-create-runtime-bundle/oci-create-runtime-bundle.1.md" -out "oci-create-runtime-bundle.1"
```

Comment:
You can use a [static pattern rules][1] and [secondary expansion][2] to avoid the phony:

    .SECONDEXPANSION:
    $(MAN): %.1: cmd/$$*/$$*.1.md
      go-md2man -in "$<" -out "$@"


[1]: https://www.gnu.org/software/make/manual/html_node/Static-Usage.html#Static-Usage
[2]: https://www.gnu.org/software/make/manual/html_node/Secondary-Expansion.html#Secondary-Expansion-of-Static-Pattern-Rules